scripts/release/publish-commands/publish-to-npm.js

```diff
@@ -54,23 +53,24 @@ const push = async ({cwd, dry, version}) => {
         // Update the @next tag to also point to it (so @next doens't lag behind).
         if (!isPrerelease) {
           await execUnlessDry(
-            `npm dist-tag add ${project}@${packageVersion} next`
+            `npm dist-tag add ${project}@${packageVersion} next`,
+            {cwd: path, dry}
           );
         }
       }
     } catch (error) {
-      errors.push(error.message);
+      errors.push(error.stack);
```

Comment:
Maybe keep both? We can just collect `error` instances and then `toString()` them, no?

Reply:
Node `error.stack` includes the message, so it's not needed

Reply:
oh kay